resume.html

```diff
@@ -5,6 +5,20 @@
     <link rel="stylesheet" type="text/css" href="css/style.css">
   </head>
   <body>
-    
+    <ul class="nav">
+  		<li><a href="index.html">About Me</a></li>
+  		<li><a href="resume.html">Resume</a></li>
+  	</ul>
+  	<div class="content">
+      <h1>Resume</h1>
+  		<h2>So you want to know what I've been up to, eh?</h2>
```

Comment:
You definitely got lazy with your spacing on this page. It could be a lot better. https://www.smashingmagazine.com/images/principles-keep-code-clean/CleanCode-3.png